### DISCUSSION TITLE
HomeAssistant Add-on: HM-MOD-RPI-PCB funktioniert nicht

### DISCUSSION THREAD

**Discussion by KarlKaracho** [OpenCCU/OpenCCU#1715] [🇩🇪 Fragen&Antworten] [answered]
Ich möchte die RasberryMatic als Addon von Home Assistant nutzen.

Habe ein HM-MOD-RPI-PCB verbaut und das auch meines Wissens nach richtig installiert.

Das HM-MOD-RPI-PCB Modul wir im Addon erkannt, und wird auch in der WEB UI von RaspberryMatic unter homematicIP-RF (HmIP) Info und HomeMatic-RF (HmRF) Info angezeigt.
Nur kann ich keine Geräte anlernen.

Da bekomme ich die Meldung:
Es wurde keinerlei nutzbares Homematic Funkmodul/LAN - Gateway für die homematic oder homematicIP Kommuniaktion im System gefunden.

Meines erachtens habe ich die Anleitung auch durchgeschaut, aber da finde ich nichts, oder bin... wieder... zu doof.

Ich habe diesen Beitrag gefunden:
https://github.com/jens-maus/RaspberryMatic/issues/1448
Ist das das gleiche Problem?

Ich habe auch noch diesen Beitrag gefunden:
https://homematic-forum.de/forum/viewtopic.php?t=69982&p=682561

Habe auch mal bei ELV gefragt, die meinen ich könnte das Modul gerne einschicken zum prüfen... hmm...

Haltet ihr das für realistisch, dass das Modul kaputt ist, obwohl es richtig angezeigt wird?

Kann mir da einer weiter helfen?

**Comment by jens-maus** [comment]
> Das HM-MOD-RPI-PCB Modul wir im Addon erkannt, und wird auch in der WEB UI von RaspberryMatic unter homematicIP-RF (HmIP) Info und HomeMatic-RF (HmRF) Info angezeigt. Nur kann ich keine Geräte anlernen.
> 
> Da bekomme ich die Meldung: Es wurde keinerlei nutzbares Homematic Funkmodul/LAN - Gateway für die homematic oder homematicIP Kommuniaktion im System gefunden.

Also diese beiden Sätze beißen sich irgendwie. Du sagst einerseits in der WebUI von RaspberryMatic wird "unter homematicIP-RF" das HM-MOD-RPI-PCB angezeigt, dann aber wird beim Anlernmodus das Fenster "Es wurde keinerlei nutzbares..." gezeigt.

Bitte daher etwas konkreter werden, screenshots zeigen und auch mal einen Blick in die `/var/hm_mode` Datei werfen lassen oder einen Screenshot der Hilfeseite in der WebUI zeigen damit man sich hier selbst ein Bild davon machen kann.

**Comment by KarlKaracho** [comment]
Hi Jens,
Danke für deine schnelle Antwort.
Ja genau das finde ich ja auch komisch :-)

Hier die Hilfe aus der Web UI
![Screenshot 2022-02-11 223417](https://user-images.githubusercontent.com/59807454/153674312-541f696d-6e38-48b9-a369-c6f9143a9b58.jpg)

Hier das Protokoll des RaspberryMatic Addons
![Screenshot 2022-02-11 223459](https://user-images.githubusercontent.com/59807454/153674433-a207d0ff-936c-4ca6-9c74-0b65007c9d07.jpg)

Und hier ein Screenshort der hm_mode Datei, wenn ich das richtig verstehe.
Bitte um Nachsicht, im Vergleich zu dir bin ich glaube ich der ultra Neuling :-)
Habe bei Port 22222 vi /var/hm_mode eingegeben
![Screenshot 2022-02-11 224052](https://user-images.githubusercontent.com/59807454/153674491-4d6d84d1-fab6-4376-8527-75434c7f5e0f.jpg)

**Comment by jens-maus** [comment, marked as answer]
Du scheinst offensichtlich die aktuellste stabile Version des Add-ons zu verwenden. Aktuell musst du für die korrekte Funktion jedoch den "Gesicherten Modus" in der HA Add-on Konfig ausschalten. Das wird sich mit der nächsten Version nicht mehr notwendig sein. Aber aktuell musst du diesen Modus in den RaspberryMatic Add-on Settings ausschalten.

**Comment by KarlKaracho** [comment]
Hey,
und wieder danke für die schnelle Antwort.

Ähm, ich bin ja nicht sicher ob ich vielleicht echt zu doof bin.
Aber welcher Protection Mode / Gesicherter Modus?

Habe auch hier gelesen
https://homematic-forum.de/forum/viewtopic.php?t=72373&p=703924

Müsste der nicht hier sein?!
![Screenshot 2022-02-11 224052](https://user-images.githubusercontent.com/59807454/153677779-b4f446c7-f921-4abc-9bb5-bc23a3d8fdf2.jpg)

**Reply by jens-maus** [reply]
Nein, der hat nix mit der Konsole zu tun. Geh in die Weboberfläche von HomeAssistant in die Addon Einstellungsseite wo man auch "Start on boot" usw einstellt. Da müsste es was mut "Gesicherter Modus" geben. Den Ausschalten und dann sollte es gehen. Zur Not das Addon komplett deinstallieren + komplett neu installieren und dann  den Modus abschalten.

**Comment by KarlKaracho** [comment]
Hehe,
mist sorry :-)

Klar das ist der falsche Screenshot.

Der sollte es sein.
![Screenshot 2022-02-11 231246](https://user-images.githubusercontent.com/59807454/153679476-59bfcd59-5275-4b1e-a823-19f57be2748a.jpg)

**Reply by jens-maus** [reply]
Sagte doch: komplett nochmal deinstallieren, im addon-store auf "updates suchen" gehen und dann das addon frisch installieren. Dann sollte der button zum
Abschalten des gesicherten modus auftauchen.

**Comment by KarlKaracho** [comment]
Habe ich in der Zwischenzeit auch gemacht.
Dann ist der Button natürlich da.

Jetzt klappt es auch.

Danke für deine Hilfe!
